ci-src-requirements.txt

```diff
@@ -1,2 +1 @@
-git+http://github.com/enthought/traits.git#egg=traits
-git+http://github.com/enthought/pyface.git#egg=pyface
+git+http://github.com/enthought/pyface.git@release/7.0#egg=pyface
```

Comment:
pyface's master has the `Union` removed, so this can be changed back to master if needed. Maybe in a separate PR so the CI for this release PR is targeting pyface 7.0?

Reply:
Yes - I am planning a separate PR after this is merged.  There is already one for Pyface.